key.go

```diff
@@ -254,7 +254,7 @@ func (k *Key) UnmarshalJSON(data []byte) error {
 //   RandomKey()
 //   NewKey("/f98719ea086343f7b71f32ea9d9d521d")
 func RandomKey() Key {
-	return NewKey(strings.Replace(uuid.NewV4().String(), "-", "", -1))
+	return NewKey(strings.Replace(uuid.Must(uuid.NewV4()).String(), "-", "", -1))
```

Comment:
I think using `Must` here is not the best idea as it will lead to panic in case of low entropy. Instead we might want to retry and/or wait+retry.

Reply:
Doesn't that only apply to e.g. embedded devices? We should probably do that anyways but, IIRC, the only difference is that the old constructor paniced internally.

Reply:
It would be also a problem on multi-tenant servers (high entropy usage) or on SSL terminating servers (with high tragic) without Haveged.